doc/sdk-cli-reference.md

```diff
@@ -9,16 +9,20 @@ Usage:
 
 ### Args
 
-* image - is the container image to be built, e.g. "quay.io/example/operator:v0.0.1". This image will be automatically set in the deployment manifests.
+* image - is the container image to be built, e.g. "quay.io/example/operator:v0.0.1".
 
 ### Flags
-
+* `--enable-tests` bool - enable in-cluster testing by adding test binary to the image
+* `--namespaced-manifest` string - path of namespaced resources manifest for tests (default "deploy/operator.yaml")
+* `--test-location` string - location of tests (default "./test/e2e")
 * `-h, --help` - help for build
 
+
 ### Use
 
-The operator-sdk build command compiles the code, builds the executables,
-and generates Kubernetes manifests. After build completes, the image would be built locally in docker. Then it needs to be pushed to remote registry.
+The operator-sdk build command compiles the code and builds the executables. After build completes, the image is built locally in docker. Then it needs to be pushed to a remote registry.
+
+If `--enable-tests` is set, the build command will also build the testing binary and add it to the docker image.
```

Comment:
Mention that this also creates an example `deploy/test-pod.yaml` that you can use to run the image as a test-pod.